specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "AttestationManagementClient",
+        "description": "Various APIs for managing resources in attestation service. This primarily encompasses per-tenant instance management.",
+        "version": "2018-09-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/providers/Microsoft.Attestation/operations": {
+            "get": {
+                "tags": [
+                    "Operations"
+                ],
+                "operationId": "Operations_List",
+                "description": "Lists all of the available Azure attestation operations.",
+                "x-ms-examples": {
+                    "Operations_List": {
+                        "$ref": "./examples/Operations_List.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                },
+                "responses": {
+                    "200": {
+                        "description": "List all the operations.",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    },
+                    "default": {
+                        "description": "Attestation service error result",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResult"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Attestation/attestationProviders/{providerName}": {
```

Comment:
Collection list apis:
`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Attestation/attestationProviders`
and 
`/subscriptions/{subscriptionId}/providers/Microsoft.Attestation/attestationProviders`
are missing.

Reply:
has been added